src/theory/arith/theory_arith_private.cpp

```diff
@@ -1923,7 +1924,8 @@ void TheoryArithPrivate::outputConflicts(){
       ++conflicts;
       Debug("arith::conflict") << "d_conflicts[" << i << "] " << conflict
                                << " has proof: " << hasProof << endl;
-      PROOF(if (d_containing.d_proofRecorder && confConstraint->hasFarkasProof()
+#if 0
```

Comment:
Delete?  @alex-ozdemir  may know best about this.  Wondering if this is required on proof-new.